float64 centerX
float64 centerY
float64 theta
float64 steering_angle
float64 velocity